django/core/management/commands/dumpdata.py

```diff
@@ -63,6 +63,10 @@ def add_arguments(self, parser):
             '-o', '--output',
             help='Specifies file to which the output is written.'
         )
+        parser.add_argument(
+            '--allow-unicode', action='store_false', dest='allow_unicode',
```

Comment:
I think it should be `store_true` and `dest` is redundant.

```suggestion
            '--allow-unicode', action='store_true',
```

Wonder if a command test should be added to test that?

Reply:
Thanks @charettes,
I've addressed your suggestion and added requested tests as well.